Dockerfile.release

```diff
@@ -10,11 +10,11 @@ RUN set -xe && \
     rm -r /root/.cache
 
 ARG ICLOUDPD_VERSION
-ADD icloudpd-latest-py2.py3-none-any.whl /tmp
+COPY dist/* /tmp
```

Comment:
The only issue with this change is that `dist/*` can contain multiple older versions, so this COPY command will include all of the older versions inside the `/tmp` folder in the Docker image. That's the reason why I was copying the latest specific version into the root folder, just to be careful about space and keep the Docker image as small as possible. But this can also be fixed by deleting all files in the `dist/*` folder before building, so I will update the build script to do that

Reply:
`/dist` is ignored in git and removed in `scripts\clean`, so this concern has been addressed before this change